lang/cs/Org.Apache.REEF.Common/metrics/Api/IMetricsSystem.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.Common.metrics.Api
+{
+    public interface IMetricsSystem
+    {
+        /// <summary>
+        /// (Re)Start the whole Metrics system.
```

Comment:
Yes second time is going to be a restart. I am not sure if somebody calls restart without  calling stop (two successive starts) if we should throw error.